--- base_worker.py	2020-05-27 23:26:54.711472336 +0200
+++ base_worker2.py	2020-05-27 23:26:20.591537522 +0200
@@ -18,6 +18,7 @@

 from copy import copy

+from botocore.client import Config
 from botocore.session import Session

 from ..core import async_traceback
@@ -55,6 +56,8 @@
             self.task_list)

     def _fix_endpoint(self):
+        # FIXME: AttributeError: 'Endpoint' object has no attribute 'timeout'
+        return
         timeout = self.client._endpoint.timeout

         # newer versions of botocore create a timeout tuple of the form (connect_timeout, read_timeout)
@@ -85,7 +88,8 @@
             return self._client
         except AttributeError:  # create a new client
             self._client = self._session.create_client(
-                service_name='swf', region_name=self._aws_region)
+                service_name='swf', region_name=self._aws_region,
+                config=Config(connect_timeout=50, read_timeout=70))
             return self._client

     @property